Functions/src/FunctionsArrayRemoveDuplicates/ArrayDuplicateRemoval.java

```diff
@@ -0,0 +1,48 @@
+package FunctionsArrayRemoveDuplicates;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+
+public class ArrayDuplicateRemoval {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please enter the length of the array:");
+        int x = sc.nextInt();
+        int[] arr = getArray(x);
+        System.out.println("Your array is: " + Arrays.toString(arr));
+
+        int arrNoDuplicates[] = removeDuplicates(arr);
+        System.out.println("The array with no duplicates is: " + Arrays.toString(arrNoDuplicates));
+    }
+
+    public static int[] getArray(int x) {
```

Comment:
Functia ar fi putut sa includa si afisarea mesajului de introducere a lungimii array-ului, cat si citirea acesteia. In acest fel, cele doua operatiuni care contribuie la citirea array-ului de la tastatura se fac impreuna,